conf/cb-event-forwarder.example.ini

```diff
@@ -2,526 +2,529 @@
+        # client_key=/etc/cb/integrations/event-forwarder/client-key.pem
+        # client_cert=/etc/cb/integrations/event-forwarder/client-cert.pem
+
+        # Uncomment authorization_token to place a value in the outgoing HTTP "Authorization" header
+        #  (used in HTTP Basic Authentication). See https://en.wikipedia.org/wiki/Basic_access_authentication
+        #  for more information. By default no Authorization header is sent.
+        # authorization_token=Basic QWxhZGRpbjpPcGVuU2VzYW1l
+
+        splunk:
+        # Uncomment ca_cert to specify a file containing PEM-encoded CA certificates for verifying the peer server
+        # ca_cert=/etc/cb/integrations/event-forwarder/ca-certs.pem
+
+        # Uncomment server_cname to force a specific server Common Name when validating the peer server
+        # certificate. Useful for situations where the peer server is using a self-signed certificate with
+        # a generic name (for example, QRadar's default certificate uses "*" as the CN)
+        # server_cname=peer.server.name
+
+        # Uncomment tls_verify and set to "false" in order to disable verification of the peer server certificate
+        # tls_verify=false
+
+        # Uncomment insecure_tls to allow connections to servers that only support TLS versions less than 1.2.
+        # The default is to only support TLSv1.2 cipher suites. This option will allow connections to TLSv1.0 and 1.1
+        # servers as well.
+        # insecure_tls=true
+
+        # Uncomment client_key and client_cert and set to files containing PEM-encoded private key and public
+        #  certificate when using client TLS certificates
+        # client_key=/etc/cb/integrations/event-forwarder/client-key.pem
+        # client_cert=/etc/cb/integrations/event-forwarder/client-cert.pem
+
+        # Set the default timeout period in seconds. By default, cb-event-forwarder will contact the remote service every five
+        #  minutes (300 seconds)
+        # bundle_send_timeout=300
+
+        # Send empty updates? By default, cb-event-forwarder will send an empty update every bundle_send_timeout seconds.
+        #  if this is set to false, then the cb-event-forwarder will not initiate a connection to the remote service unless
+        #  there are events to send.
+        # upload_empty_files=true
+
+        # Set the maximum file size before the events must be flushed to the remote service. The default is 10MB.
+        # bundle_size_max=10485760
+        #HEC TOKEN
+        #
+        #hec_token stores the HEC token to be used when communicating with splunk
+        #
+        hec_token: PASSWORD
+
+
+        #CbR Ef 4.0 supports event filtering and custom encoding using golang's
+        # text/template package / templating language
+        # Filter templates  must return DROP or KEEP strings
+        # User can define filters that operate on the keys/values in the in-flight CBEventMessage
+        filter:
+            enabled: true
```

Comment:
The filters and templates should be disabled by default in the example configuration file, since most customers will just take the example and run with it (for better or worse)